app/controllers/finders_controller.rb

```diff
@@ -37,4 +38,8 @@ def facet_params
   def keywords
     params[:keywords] unless params[:keywords].blank?
   end
+
+  def atom_url
+    request.original_url.gsub('.json', '').gsub(finder_slug, "#{finder_slug}.atom")
```

Comment:
The `atom` URL is just the regular URL with `.atom` added before the query string. Since we update the URL each time the filters are updated, what's the benefit of doing this on the server and sending it in the response rather than doing it on the client?